Scripts/get_deployed_commit.py

```
@@ -96,20 +99,33 @@ def main():
+    # refs with 'git fetch origin <ref>', and GitHub only serves unadvertised objects by full SHA.
+    # The 'sha' media type returns the 40-character hash as plain text.
     if len(commit) < 40:  # Full SHA-1 hash is 40 characters
         print(f"Attempting to translate short commit hash {commit} to full commit hash")
         try:
             request = urllib.request.Request(
-                f"{REPO_COMMIT_ROOT}{commit}",
-                headers={'Accept': 'application/json'}
+                f"{REPO_COMMITS_API_ROOT}{commit}",
+                headers={'Accept': 'application/vnd.github.sha'}
             )
             with urllib.request.urlopen(request) as response:
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)